### PR TITLE
Div 8568

### PR DESCRIPTION
**Pull request by varadeth**
@prasanna-egov @egov-joy @saiprakash-v Please review

### CODE REVIEW

File: backend/vc-certification-service/src/routes/certificate.route.js [hunk 1]
```diff
@@ -6,5 +6,6 @@ const router = express.Router();
 
 router.post(`/certify/:entityType`, authMiddleware, certificateController.createCertificate)
 router.get(`/certificate/:entityName/:certificateId`, authMiddleware, certificateController.getCertificate)
+router.put(`/certify/:entityName/:entityId`, authMiddleware, certificateController.updateCertificate);
```
Comment (prasanna-egov) [inline backend/vc-certification-service/src/routes/certificate.route.js:9]: I know that entityID and certificateID are the same, but can we keep the same nomenclature?

Comment (varadeth) [reply backend/vc-certification-service/src/routes/certificate.route.js:9]: sure. same nomenclature is good thing to be followed. will update this

File: backend/vc-certification-service/envVars.js [hunk 2]
```diff
@@ -0,0 +1 @@
+process.env.SUNBIRD_REGISTRY_URL = '';
```
Comment (egov-joy) [inline backend/vc-certification-service/envVars.js:1]: If this is being used only for the unit tests, can we rename the file to indicate this and so it is clear that it is not being used by the service as such for its functionality.

Comment (varadeth) [reply backend/vc-certification-service/envVars.js:1]: ok

**Review by egov-joy — APPROVED** [review]

**Review by prasanna-egov — APPROVED** [review]

**Pull request merged**